components/sysutils/ipmitool/Makefile

```diff
@@ -13,31 +13,30 @@
 # Copyright 2017 Gary Mills
 # Copyright 2016 Alexander Pyhalov
 # Copyright 2019 Michal Nowak
+# Copyright 2022 Niklas Poslovski
 #
 
 PREFERRED_BITS=		64
 
```

Comment:
PREFERRED_BITS is not necessary if the package is only being build in one variant.

Reply:
Should I add BUILD_BITS instead or is 64bit default now so that I can completely remove the line?

Reply:
BUILD_BITS=64 is the new default.